src/components/ContentWidgets/ToDoList/TodoList.tsx

```diff
@@ -0,0 +1,28 @@
+
+import React from 'react';
+import { ContentSingularData } from '../../_data/ContentSingularData';
+import './TodoStyle.css';
+
+
+export type TodoItem = {
+	name: string
+	check: string // checked | unchecked
```

Comment:
This is an unnecessary comment following this. Instead, you will want to make this variable into something human understandable, like `completed` instead of `check`.